build-utils/protocol-base-mspec/src/test/resources/mspec.example

```diff
@@ -208,9 +208,9 @@
 
 [discriminatedType 'ADSData' [CommandId 'commandId', bit 'response']
     [typeSwitch 'commandId', 'response'
-        ['CommandId.INVALID', 'true' AdsInvalidResponse]
-        ['CommandId.INVALID', 'false' AdsInvalidRequest]
-        ['CommandId.ADS_READ_DEVICE_INFO', 'true' AdsReadDeviceInfoResponse
+        ['INVALID', 'true' AdsInvalidResponse]
```

Comment:
Think @chrisdutz talked about that: This is still a reference to a existing enum not a string right? Like you don't need to qualify them here